docs/content/doc/upgrade/from-gitea.en-us.md

```diff
@@ -0,0 +1,85 @@
+---
+date: "2021-09-02T16:00:00+08:00"
+title: "Upgrade from an old Gitea"
+slug: "upgrade-from-gitea"
+weight: 10
+toc: false
+draft: false
+menu:
+  sidebar:
+    parent: "upgrade"
+    name: "From Gitea"
+    weight: 10
+    identifier: "upgrade-from-gitea"
+---
+
+# Upgrade from an old Gitea
+
+**Table of Contents**
+
+{{< toc >}}
+
+Gitea provides automatically upgrade mechanism. Just get a new Gitea, stop the old one, run the new one.
+Everytime a Gitea instance runs, it checks whether an upgrade action should be taken. 
+If an upgrade action is required, Gitea will take some time to complete the upgrade and then serve.
+
+## Downgrade and backup
+
+Gitea keeps compatibility for versions whose first two fields are the same (`a.b.x` -> `a.b.y`), 
+these versions can be upgraded and downgraded with the same database structure. 
+Otherwise (`a.b.?` -> `a.c.?`), a new Gitea will upgrade an old database 
+to a new structure which may not be used by an old Gitea.
```

Comment:
```suggestion
Otherwise (`a.b.?` -> `a.c.?`), a newer Gitea version will upgrade the old database 
to a new structure that may differ from the old version.
```